providers/bitbucket/common.go

```diff
@@ -0,0 +1,34 @@
+package bitbucket
+
+import (
+	"github.com/inconshreveable/log15"
+	"github.com/src-d/rovers/core"
+	"github.com/src-d/rovers/providers/bitbucket/model"
+
+	rmodel "gopkg.in/src-d/core-retrieval.v0/model"
+)
+
+const httpsCloneKey = "https"
+
+func getMention(r *model.Repository) *rmodel.Mention {
+	aliases := []string{}
+	mainRepository := ""
+	for _, c := range r.Links.Clone {
+		if c.Name == httpsCloneKey {
+			mainRepository = c.Href
+		}
+		aliases = append(aliases, c.Href)
+	}
+	if mainRepository == "" {
+		log15.Error("no https repositories found", "clone urls", r.Links.Clone)
```

Comment:
Can we get the first alias instead of keep mainRepository empty? (Is not your fault, but we can improve this code now ;))